common/src/main/java/com/wynntils/features/user/ItemScreenshotFeature.java

```diff
@@ -106,13 +105,7 @@ private static void takeScreenshot(Slot hoveredSlot) {
         fb.bindWrite(false);
         poseStack.pushPose();
         poseStack.scale(scalew, scaleh, 1);
-        RenderUtils.drawTooltip(
-                tooltip.stream()
-                        .map(Component::getVisualOrderText)
-                        .map(ClientTooltipComponent::create)
-                        .collect(Collectors.toList()),
-                poseStack,
-                font);
+        RenderUtils.drawTooltip(poseStack, tooltip, font, true);
```

Comment:
Was this an unrelated change?

Reply:
Change of method signature.

Reply:
Behaves the same way, but the method is more generalized.